vaadin-menu-bar-flow-parent/vaadin-menu-bar-testbench/src/main/java/com/vaadin/flow/component/menubar/testbench/MenuBarItemElement.java

```diff
@@ -35,7 +35,9 @@ public class MenuBarItemElement extends TestBenchElement {
      */
     public TestBenchElement getSubMenu() {
         waitForSubMenu();
-        return getPropertyElement("__overlay");
+        TestBenchElement listBox = getPropertyElement("parentElement");
+        TestBenchElement subMenu = listBox.getPropertyElement("parentElement");
+        return subMenu.getPropertyElement("_subMenu");
```

Comment:
Does this actually work? The parent element of the list box should be the slotted wrapper element?

We could update `openSubSubMenu_clickCheckableItem_checkableStateChanges` to use this instead of `getAllSubMenus` to keep it tested.